avm/res/network/connection/tests/e2e/defaults/main.test.bicep

```
@@ -24,6 +24,7 @@ param password string = newGuid()
 @description('Optional. A token to inject into the name of each resource.')
 param namePrefix string = '#_namePrefix_#'
 
+#disable-next-line no-hardcoded-location // Just a value to avoid ongoing capaity challenges
```

Small typo here and similar occurrences
```suggestion
#disable-next-line no-hardcoded-location // Just a value to avoid ongoing capacity challenges
```

---

Fixed for all